aws/eks_utils_for_test.go

```diff
@@ -0,0 +1,106 @@
+package aws
+
+import (
+	"fmt"
+	"math/rand"
+	"testing"
+	"time"
+
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/eks"
+	"github.com/aws/aws-sdk-go/service/iam"
+	gruntworkerrors "github.com/gruntwork-io/gruntwork-cli/errors"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// getRandomEksSupportedRegion - Returns a random AWS region that supports EKS.
+// Refer to https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/
+func getRandomEksSupportedRegion() string {
+	supportedRegions := []string{
+		"us-east-1",
+		"us-east-2",
+		"us-west-2",
+		"eu-west-1",
+	}
+	rand.Seed(time.Now().UnixNano())
+	randIndex := rand.Intn(len(supportedRegions))
+	return supportedRegions[randIndex]
```

Comment:
There's a terratest `GetRandomRegion` helper that takes in a whitelist and/or blacklist of regions you can use...